.travis.yml

```diff
@@ -46,51 +46,6 @@ matrix:
       distributions: sdist
       on:
         tags: true
-  - script: scripts/test-build.sh
-    install:
-    - pip install --user pybind11 nose2
```

Comment:
@r-rojo I don't see pip dependencies ported over to the GitHub CI. Are they still needed?

Reply:
Probably not, seeing latest CI run: https://github.com/OPAE/opae-sdk/runs/930116673?check_suite_focus=true